### DISCUSSION TITLE
Middle click creates new tab

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#855] [Ideas]
### What feature would you like to see?

It would be nice to have middle click function that opens a new tab (much like vivaldi). It makes adding tabs from any space in the vertical tabs much quicker and efficient!

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/365_

**Comment by LarveyOfficial** [comment]
This would be great. plz

**Comment by SirZokxyron** [comment]
Completely agree this would be a great addition.

**Comment by baduhai** [comment]
Indeed, I miss this dearly, from Firefox.

**Comment by LAUCHGeorge** [comment]
I also miss that specific feature A LOT. Please implement this, I dont imagine it being that hard

**Comment by yotamguttman** [comment]
amazing idea! would spare me some unnecessary mouse travel to the new tab button

**Comment by neogeographica** [comment]
Yes please, either middle click or else double click as per #1300 .

**Comment by kristijanribaric** [comment]
Implemented in the version `1.0.1-a.8`
